fineract-provider/src/test/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImplTest.java

```diff
@@ -75,15 +74,15 @@ public void setup() {
     @Test
     public void testSqlInjectionCaughtQueryDataTable() {
         mockDatatableValidation();
-        assertThrows(SQLInjectionException.class, () -> {
+        assertThrows(PlatformApiDataValidationException.class, () -> {
```

Comment:
Are you sure this is correct?

Reply:
Yes, because column named "1; DROP TABLE m_loan; SELECT" does not exist in the table.